### ISSUE TITLE
no such column: result.resource_kind

### ISSUE THREAD

**Issue by jemag** [fjogeleit/policy-reporter#296]
Specifying a `namespace` then a `kind` in PolicyReports:

![image](https://user-images.githubusercontent.com/7985687/230472594-8ead6fe6-3f79-45a9-95b9-7cbed85451e0.png)

 makes a call to `/api/v1/namespaced-resources/status-counts?namespaces=external-secrets&kinds=Deployment&sources=kyverno` which results in HTTP 500 and error 
```json
{ "message": "no such column: result.resource_kind" }
```

Same thing in policy-reporter logs:
```
1.6808078990585613e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808079024404042e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808079134729822e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808079234866688e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.680807933318161e+09   error   failed to load data     {"error": "no such column: result.resource_kind"}
1.680807945024263e+09   error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808079556578388e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808079662919326e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.680807976505904e+09   error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808079873555405e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.680807997470817e+09   error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808080074714196e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808080175691354e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808080281111572e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.680808038116923e+09   error   failed to load data     {"error": "no such column: result.resource_kind"}
1.680808048125585e+09   error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808080587587888e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808080692280343e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808080795020645e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.68080809012405e+09    error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808081009148939e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808081112028785e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.680808121488998e+09   error   failed to load data     {"error": "no such column: result.resource_kind"}
1.680808132016841e+09   error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808081420348692e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808081520528793e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808081623261924e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808081723301578e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808081825071287e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808081929861999e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.680808203637531e+09   error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808082144337492e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}
1.6808082251449366e+09  error   failed to load data     {"error": "no such column: result.resource_kind"}

```

**Comment by fjogeleit** [comment]
hey, thanks for reporting. I'll have a look on it

**Comment by fjogeleit** [comment]
I published the image tag `2.14.3` with the fix. You can just override `image.tag` with `2.14.3` until its released.
